package.json

```diff
@@ -6,6 +6,7 @@
     "@blueprintjs/core": "~3.18.1",
     "@blueprintjs/datetime": "~3.13.0",
     "@blueprintjs/icons": "~3.10.0",
+    "@material-ui/core": "^4.5.0",
```

Comment:
On one hand you add depdency to `@material-ui/core` and on the other hand, you add some `// eslint-disable-line import/no-extraneous-dependencies` linting exception ;)
We have a choice to make!

I would personaly prefer not setting `@material-ui/core` as project dependency, as It imply sticking to a specific version, while the sole hard dependency is through react-admin. In case of upgrade of react-admin, it could result in a *desync* of version between our and the one required by react-admin. It's not a serious issue, but that was the starting point of not defining this dependency until now.

For now I do not know what is the best solution. But I'm sure that we should standardize in only one of these way:
- Keep eslint exception and not adding MUI as dependency, or:
- Keep dependency to MUI and remove all eslint exception that are concernd